keyboards/nopunin10did/railroad/config.h

```diff
@@ -0,0 +1,53 @@
+// The Railroad Keyboard
+// Copyright 2020 by W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
+// Licensed under GNU GPL v2, as per the QMK firmware project
+
+#ifndef CONFIG_H
+#define CONFIG_H
```

Comment:
```suggestion
#pragma once
```

Prefer `#pragma once` to these old-style include guards.